.github/workflows/nightly.yml

```diff
@@ -200,46 +200,46 @@ jobs:
             echo "ARTIFACTS_EXIST=true" >> $GITHUB_OUTPUT
           fi
       - name: Yarn- Install Dependencies
-        if: ${{ ! contains(github.event.head_commit.message, 'Bump metro@') && steps.check_if_apple_artifacts_are_there.outputs.ARTIFACTS_EXIST != true }}
```

Comment:
Why'd we drop this test?

Reply:
because we never had such a check in the CircleCI implementation. I don't know why or who added that... and actually it makes no sense. If we have to run a JS script - and we need if there is no cache, because we use a JS script to create the tarball - we need to install the dependencies.